documentation/healthcare-analytics-workspace.adoc

```diff
@@ -0,0 +1,471 @@
+====
+
+[%collapsible]
+.Reveal the solution
+====
+.Number of reactions reported by the Consumer
+[source,cypher]
+----
+MATCH (rs:ReportSource)<-[rb:REPORTED_BY]-(c:Case)-[:HAS_REACTION]->(r:Reaction)
+WHERE rs.name = "Consumer"
+RETURN r.description as reaction, count(r) as totalReactions 
+ORDER BY totalReactions DESC
+----
+====
+
+In the next section, you will use the `Outcome` node and `RESULTED_IN` relationship to find the outcome of the cases.
+
+== Outcomes
+
+The outcome of a case is stored as an `Outcome` node. The `RESULTED_IN` relationship connects the `Case` to the `Outcome`.
+
+Review the following query and try to predict the results before running it.
+
+[source,cypher]
+----
+MATCH (c:Case)-[:RESULTED_IN]->(o:Outcome)
+MATCH (c)-[:IS_PRIMARY_SUSPECT]->(d)
+RETURN d.name as drug, o.outcome as outcome, count(c) as cases
+ORDER BY drug
+----
+
+[%collapsible]
+.Reveal the answer
+====
+The query returns the number of cases for each outcome for each drug.
+====
+
+[NOTE]
+.Challenge
+====
+Can you create a query to return the top reactions where the outcome of the case was `Death`?
+====
+
+[%collapsible]
+.Reveal the solution
+====
+.Top reactions where the outcome was Death
+[source,cypher]
+----
+MATCH (c:Case)-[:RESULTED_IN]->(o:Outcome)
+MATCH (c)-[:HAS_REACTION]-(r:Reaction)
+WHERE o.outcome = 'Death'
+RETURN r.description as reaction, count(c) as cases
+ORDER BY cases DESC
+----
+====
+
+In the next section, you will use the `AgeGroup` node to classify cases by age.
+
+== Age Groups
+
+Each case has an `AgeGroup` node that contains the age group of the person involved in the case. 
+The `FALLS_UNDER` relationship connects the `Case` to the `AgeGroup`.
+
+This query returns the number of cases for each age group.
+
+.Cases by age group
+[source,cypher]
+----
+MATCH (a:AgeGroup)<-[:FALLS_UNDER]-(c:Case)
+MATCH (c)-[:HAS_REACTION]->(r)
+RETURN a.ageGroup as ageGroupName, count(r) as total
+ORDER BY total DESC
+----
+
+Using this data you can also look at the outcomes for each age group.
+For example, the following query returns the number of cases which resulted in `Death` for each age group.
+
+.Deaths by age group
+[source,cypher]
+----
+MATCH (a:AgeGroup)<-[:FALLS_UNDER]-(c:Case)
+MATCH (c)-[:RESULTED_IN]->(o:Outcome{outcome:"Death"})
+RETURN a.ageGroup as ageGroupName, count(c) as total
+ORDER BY total DESC
+----
+
+By comparing total number of reactions and the number of reactions by age group you can create a percentage of reactions for each age group.
+
+.Percentage of side effects for each age group
+[source,cypher]
+----
+MATCH (c:Case)-[:HAS_REACTION]->(r)
+WITH count(r) as totalReactions
+MATCH (a:AgeGroup)<-[:FALLS_UNDER]-(c)-[:HAS_REACTION]->(r)
+WITH a, count(r) as ageGroupWiseReactions, totalReactions
+RETURN a.ageGroup as ageGroupName, (ageGroupWiseReactions*100.00)/totalReactions as perc
```

Comment:
Does this warrant a bit of an explanation of what's going on?

Reply:
Thank you. I have gone through your comments and made the change :)